.travis.yml

```diff
@@ -43,7 +42,6 @@ script:
 
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
-  - if [ $(phpenv version-name) = "5.5" ] && [ "$REMOVE_XDEBUG" = "0" ]; then php ocular.phar code-coverage:upload --format=php-clover tests/clover.xml --revision=$TRAVIS_COMMIT; fi
```

Comment:
And there is deleting PHP 5.5 from composer?
https://github.com/getsentry/sentry-php/pull/473/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L22